File: 900/1471A/main.java
import java.util.*;
public class main
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        
        while(t-- > 0)
        {
            int n = sc.nextInt();
            int x = sc.nextInt();
            
            long sum1 = 0;
            long sum2 = 0;
            
            for(int i=0;i<n;i++)
            {
                int a = sc.nextInt();
                sum1 += a;
                
                sum2 += a/x;
                if(a%x != 0)
                    sum2++;
            }
 
            if(sum1%x != 0)
                sum1 = sum1/x + 1;
            else
                sum1 = sum1/x;
            
            System.out.println(sum1+" "+sum2);
        }
    }
}

File: 900/1471A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int t;
    cin >> t;

    while (t--) 
    {
        int n, x;
        cin >> n >> x;

        long long sum1 = 0, sum2 = 0;

        for (int i = 0; i < n; i++) 
        {
            int a;
            cin >> a;
            sum1 += a;

            sum2 += a / x;
            if (a % x != 0) 
                sum2++;
        }

        // Compute minimum possible sum (ceil(sum1 / x))
        long long minSum;
        if (sum1 % x != 0)
            minSum = sum1 / x + 1;
        else
            minSum = sum1 / x;

        cout << minSum << " " << sum2 << "\n";
    }

    return 0;
}
